src/cdk/app.ts

```diff
@@ -0,0 +1,53 @@
+import apigateway = require('@aws-cdk/aws-apigateway');
+import lambda = require('@aws-cdk/aws-lambda');
+import s3 = require('@aws-cdk/aws-s3');
+import cdk = require('@aws-cdk/core');
+
+export class LambdaService extends cdk.Stack {
+	constructor(scope: cdk.Construct, id: string) {
+		super(scope, id, { env: { region: 'eu-west-1' } });
+
+		const stage = new cdk.CfnParameter(this, 'Stage', {
+			type: 'String',
+			default: 'CODE',
+		});
+
+		const bucket = 'playground-dist';
+		const key = `playground/${stage.value}/lambda/lambda.zip`;
+
+		const handler = new lambda.Function(this, 'playground-newsletter-api', {
+			runtime: lambda.Runtime.NODEJS_12_X,
+			code: lambda.Code.fromBucket(
+				s3.Bucket.fromBucketName(this, 'lambda-code-bucket', bucket),
+				key,
+			),
+			handler: 'server.handler',
+			functionName: `playground-newsletter-api-${stage.value}`,
+		});
+
+		// If you need access to parameter store then uncomment and adjust the following:
+		// handler.addToRolePolicy(
+		//     new iam.PolicyStatement({
+		//         effect: iam.Effect.ALLOW,
+		//         resources: ['arn:aws:ssm:eu-west-1:[your account ID goes here]:parameter/*'],
+		//         actions: ['ssm:GetParameter'],
+		//     }),
+		// );
```

Comment:
Needed? As in, do we expect this use case to happen in future or are there already docs for this elsewhere and we don't need to keep this here